.github/workflows/run-forge.yaml

```diff
@@ -17,10 +17,6 @@ on:
         type: string
         description: The Forge k8s namespace to be used for test. This value should manage Forge test concurrency. It may be truncated.
 
-# temporarily limit amount of concurrent forge runs until we have autoscaling forge
-concurrency:
-  group: ${{ inputs.merge_or_canary }}
```

Comment:
removing this since it's not required anymore now that we have a sorta autoscaling forge